Major Cleanup of Baseboard Section of Engineering Reference

RKStrand opened the pull request:
Pull request overview
---------------------
Significant cleanup, corrections, reorganization, etc. of the baseboard documentation in the Engineering Reference.  Lots of changes, many rewrites.  Things are now standardized among the baseboard models, language clarified, equations checked and corrected.  This section is much better than it was.  This work was what GitHub Issue #3625 became.

### Work Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ x ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ x ] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
   - Defect: This pull request repairs a github defect issue #3625 

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [ ] Code style (parentheses padding, variable names)
 - [ ] Functional code review (it has to work!)
 - [ ] If defect, results of running current develop vs this branch should exhibit the fix
 - [ ] CI status: all green or justified
 - [ ] Performance: CI Linux results include performance check -- verify this
 - [ ] Unit Test(s)
 - C++ checks:
   - [ ] Argument types
   - [ ] If any virtual classes, ensure virtual destructor included, other things
 - IDD changes:
   - [ ] Verify naming conventions and styles, memos and notes and defaults
   - [ ] Open windows IDF Editor with modified IDD to check for errors
   - [ ] If transition, add rules to spreadsheet
   - [ ] If transition, add transition source
   - [ ] If transition, update idfs
 - [ ] If new idf included, locally check the err file and other outputs
 - [ ] Documentation changes in place
 - [ ] Changed docs build successfully
 - [ ] ExpandObjects changes?
 - [ ] If output changes, including tabular output structure, add to output rules file for interfaces

RKStrand committed:
Major cleanup of baseboard docs

Significant cleanup, corrections, reorganization, etc. of the baseboard
documentation in the Engineering Reference.  Lots of changes, many
rewrites.  Things are now standardized among the baseboard models,
language clarified, equations checked and corrected.  This section is
much better than it was (hopefully).

RKStrand commented:
@mjwitte Not sure if me selecting you as a potential reviewer actually sends you an email or not, but I know that this will.  I picked you because you have been involved in the discussion about this.  I took the liberty of not just cleaning up the organization but going beyond that and giving the entire section a full review including many rewrites, edits, etc.  If you have any questions, please let me know.  If you need someone else to do this review, please let me know and I'll contact someone else or flip it to another name.  Thanks!

Myoldmopar commented on doc/engineering-reference/src/simulation-models-encyclopedic-reference-001/baseboard-heaters.tex:
```diff
@@ -130,19 +128,23 @@ \subsubsection{Simulation and Control}\label{simulation-and-control-2-000}
 \end{equation}
 
-where \({c_{p,w}}\) ~is the specific heat capacity of the water.
+where \({c_{p,w}}\) is the specific heat capacity the water (calculated based on the average water temperature).
```
*of* the water

RKStrand replied:
@Myoldmopar Another good catch--this is now also fixed.  The changes you requested should now be up in GitHub.

Myoldmopar commented on doc/engineering-reference/src/simulation-models-encyclopedic-reference-001/baseboard-heaters.tex:
```diff
@@ -1,24 +1,28 @@
 \section{Baseboard Heaters }\label{baseboard-heaters}
 
-\subsection{Hot Water Baseboard Heater with Only Convection}\label{hot-water-baseboard-heater-with-only-convection}
+There are five different baseboard models in EnergyPlus.  Two of these models are convection only while three of these models have both radiative and convective heat transfer to the zone.  The hot water and electric baseboard models with only convection provide a simple convective heat gain to the space via the baseboard unit.  The models for these convection only units are fairly simple and simp ...
```
fairly simple and ~~simply~~ provide

RKStrand replied:
@Myoldmopar Good catch--this should now be fixed.

Myoldmopar commented on doc/engineering-reference/src/simulation-models-encyclopedic-reference-001/baseboard-heaters.tex:
```diff
@@ -130,19 +128,23 @@ \subsubsection{Simulation and Control}\label{simulation-and-control-2-000}
-$\dot{m}_{a,std} = 0.0062 + .0000275 \dot{Q}_{design}$
+\begin{equation}
+\dot{m}_{a,std} = 0.0062 + .0000275 \dot{Q}_{design}
+\end{equation}
```
:+1:

Myoldmopar reviewed (changes requested):
I got a little nitpicky, but only on a couple of lines.  It's only because this is now so much better than it was!  If you can address those two little changes I'll verify they build OK and merge it in.  Thanks @RKStrand !

RKStrand committed:
Review comments addressed

Changes made to the documents as a result of the review comments.
Comments were addressed exactly as requested.
